compiler-core/src/parse/snapshots/gleam_core__parse__tests__type_angle_generics_usage_error.snap

```diff
@@ -0,0 +1,20 @@
+---
+source: compiler-core/src/parse/tests.rs
+expression: "let list: List<Int> = []"
+---
+----- SOURCE CODE
+let list: List<Int> = []
+
+----- ERROR
+error: Syntax error
+  ┌─ /src/parse/error.gleam:1:15
+  │
+1 │ let list: List<Int> = []
+  │               ^ I was expecting `(` here.
+
+Instead of angle brackets, Gleam uses parentheses for generic types:
```

Comment:
```suggestion
Type parameters use lowercase names and surrounded by parentheses.
```

Reply:
I've gone for

> Type parameters use lowercase names and _**are**_ surrounded by parentheses.

Hope that's alright, happy to tweak!